src/mlx_transformers/models/xlm_roberta.py

```diff
@@ -451,6 +451,235 @@ def __call__(
         )
 
 
+class XLMRobertaClassificationHead(nn.Module):
```

Comment:
I created a Mixin to allow us load directly from Huggingface without having to convert first, so we can change add it here by importing the mixin

`from .base import MlxPretrainedMixin` and then `class XLMRobertaClassificationHead(nn.Module, MlxPretrainedMixin):`

Reply:
With this we can easily do 

```
model = XLMRobertaClassificationHead(config)
model.from_pretrained(model_name)
```

Reply:
perfect, this affected the NSP I was building for Bert. So I paused it to fix the Pretrained class first